core/frontend-devtools/src/widgets/ToolSettingsTracker.ts

```diff
@@ -38,7 +38,7 @@ export class ToolSettingsTracker {
       name: "Preserve World Up When Rotating",
       id: "ts_preserveWorldUp",
       isChecked: ToolSettings.preserveWorldUp,
-      handler: (_cb) => { ToolSettings.preserveWorldUp = !ToolSettings.preserveWorldUp; IModelApp.toolAdmin.exitViewTool(); },
+      handler: (_cb) => { ToolSettings.preserveWorldUp = !ToolSettings.preserveWorldUp; void IModelApp.toolAdmin.exitViewTool(); },
```

Comment:
Is `void somethingAsync` a sneaky way around having to disable `no-floating-promises` lint rule? Here and in ViewportTools.ts.

Reply:
yes, it is what the linter/compiler (i'm never sure which) suggests now

Reply:
Way easier to overlook, way harder to grep for (as I am currently doing).

Reply:
although it is far less obvious, and impossible to search for. Maybe we shouldn't use it?

Reply:
See my prev comment - hard agree.

Reply:
changed